js/gated-content/src/components/CategoryTeaser.vue

```diff
@@ -22,7 +23,8 @@ export default {
   computed: {
     image() {
       // const vid = this.category.attributes.field_gc_category_media;
-      return '';
+      return this.category.attributes.field_media_image ?
+        this.category.attributes.field_media_image.uri.url : null;
```

Comment:
Can we use this: https://www.drupal.org/project/jsonapi_image_styles and not load original image?

Reply:
Fixed in https://github.com/fivejars/openy_gated_content/pull/20/commits/51c06b58d5fa9ae78263412ec141bda2de6a1f47